test/MinterGateway.t.sol

```diff
@@ -1532,15 +1541,17 @@ contract MinterGatewayTests is TestUtils {
         uint240 undercollateralizedPenalty = ((activeOwedM + missedUpdatePenalty) * _penaltyRate) / (2 * ONE);
 
         vm.expectEmit();
-        emit IMinterGateway.PenaltyImposed(
+        emit IMinterGateway.MissedIntervalPenaltyImposed(
             _minter1,
+            2,
             _minterGateway.getPrincipalAmountRoundedUp(missedUpdatePenalty),
             missedUpdatePenalty
         );
 
         vm.expectEmit();
-        emit IMinterGateway.PenaltyImposed(
+        emit IMinterGateway.UndercollateralizedPenaltyImposed(
             _minter1,
+            activeOwedM + missedUpdatePenalty,
```

Comment:
Shouldn't we log the `excessOwedM` amount before the penalty is imposed?

Reply:
No, since the `excessOwedM` used for the undercollateralized penalty includes the `missedUpdatePenalty`.